### PR TITLE
Add DE documentation for Multithreaded modes

### PR DESCRIPTION
**Pull request by kennym**

### CODE REVIEW

File: README.de.md [hunk 1]
```diff
@@ -2821,6 +2821,42 @@ Die Optionen sind:
 -x # Mutex-Lock einschalten (Standard ist off)
 ```
 
+### Multi-threading
+
+_Paraphrasiert von [dieser Antwort auf StackOverflow][so-answer] von Konstantin_
+
+Sinatra erlegt kein Nebenläufigkeitsmodell auf, sondern überlässt dies dem
+unterlegendem Rack-Proxy (Server), so wie Thin, Puma oder WEBrick.
```
Comment (burningTyger) [inline README.de.md]: unterlegendem?

File: README.de.md [hunk 2]
```diff
@@ -2821,6 +2821,42 @@ Die Optionen sind:
...
+Sinatra selbst ist Thread-sicher, somit ist es kein Problem wenn der
+Rack-Proxy ein anderes Threading-Modell für Nebenläufigkeit benutzt.
+Das heißt, dass wenn der Server gestartet wird, dass man die korrekte
+Aufrufsmethode benutzen sollte für den jeweiligen Rack-Proxy.
+Das folgende Beispiel ist eine Veranschaulichung eines mehrprozeßigen
```
Comment (burningTyger) [inline README.de.md]: mehrprozessigen

**Comment by kennym** [comment]
@burningTyger thanks for the review - I've updated those lines.

**Comment by kytrinyx** [comment]
Thank you!

**Pull request merged**